src/components/FetchErrorMessage.tsx

```diff
@@ -0,0 +1,38 @@
+import ErrorMessage from '@components/errorMessage';
+import { CgRedo } from '@react-icons/all-files/cg/CgRedo';
+import { ErrorMessageProps } from '@components/errorMessage/type';
+import cn from '@utils/cn';
+
+interface FetchErrorMessageProps extends ErrorMessageProps {
+  onClickRetryButton: () => void;
+  iconClass?: string;
+}
+
+function FetchErrorMessage({
+  onClickRetryButton,
+  size,
+  theme,
+  weight,
+  iconClass,
+  ...props
+}: FetchErrorMessageProps) {
+  const handleClickButton = () => onClickRetryButton();
+  return (
+    <ErrorMessage
+      size={size}
+      theme={theme}
+      weight={weight}
+      className={props.className}
+    >
+      <div className='flex w-full items-center space-x-3'>
+        <button title='재시도' onClick={handleClickButton}>
```

Comment:
여기도 함수 한번 더 생성하지 않고 그대로 사용하는게 좋을 것 같아요.
```suggestion
        <button title='재시도' onClick={onClickRetryButton}>
```